include/dlaf_c/eigensolver/eigensolver.h

```diff
@@ -0,0 +1,76 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <dlaf_c/desc.h>
+#include <dlaf_c/utils.h>
+
+#ifdef DLAF_WITH_SCALAPACK
+
+/// Eigensolver
+///
+/// The matrices $A$ and $Z$ are assumed to be distributed and in host memory.
```

Comment:
Is $A$ displaying correctly in the doc?

Reply:
I still need to build and double check the docs locally (but I think `\f` is missing).